lib/constant.ml

```diff
@@ -28,15 +28,188 @@ type offset = int
 
 let compare_tag = Misc.opt_compare String.compare
 
+module PAC = struct
```

Comment:
The file `constant.ml` is already quite large. Having the `PAC` module in its own file is an option to consider.